### PR TITLE
boards/msbiot: Updated/fixed documentation

### PR DESCRIPTION
**Pull request by maribu**
### Contribution description

- Updated outdated information
- Fixed Markdown to generate desired output
- Replaced references to source code by references to their documentation
- Correct use of units (b = bit, B = byte, Ki = 1024, K = 1000, ...)

### Testing procedure

- Check if CI detects any issues with Doxygen
- All remaining MSB-IoT Boards are now owned by our work group, so no one is really affected by this change except of us
- As long as nothing unrelated sneaked into this PR, I think little to no review/testing is required, as basically only our work group is affected.

### Issues/PRs references

None

### CODE REVIEW

File: boards/msbiot/doc.txt [hunk 1]
```diff
@@ -12,130 +12,105 @@ The MSB-IoT was developed at FU Berlin and is based on a STM32F415RG MCU.
...
+| Color | red       | yellow        | green     |
+| Pin   | PB8       | PB14          | PB15      |
 
-| NAME  | LED_RED   | LED_YELLOW | LED_GREEN |
-| ----- | --------- | ---------- | --------- |
-| Color | red       | yellow     | green     |
-| Pin   | PB8       | PB14       | PB15      |
 The LEDs can be controlled inside your RIOT application by using the LED
-macros provided [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/msbiot/include/board.h).
+macros provided in @ref boards_msbiot.
```
Comment (aabadie) [inline boards/msbiot/doc.txt]: Using this `@ref` is self referencing the board documentation page. I see 2 other possibilities:
- reference the board.h file (the link is http://doc.riot-os.org/msbiot_2include_2board_8h.html)
- directly give the macro names here.

Comment (maribu) [reply boards/msbiot/doc.txt]: Thanks for spotting that. I directly referenced the macro names now.

**Comment by maribu** [comment]
@aabadie: Thanks for your quick review and spotting the self-reference :-)

**Comment by maribu** [comment]
Hit merge? (With only a doc file being touch, rerunning Murdock before merging could be skipped, I guess.)

**Comment by aabadie** [comment]
Ah, sorry for the reminder. I thought I already merged this one...

**Pull request merged**